package.json

```diff
@@ -1,12 +1,11 @@
 {
-		"name":		   "dmx",
-		"version":	   "0.1.1",
-		"author":	   "Sebastian Wiedenroth <wiedi@frubar.net>",
+		"name": "dmx",
```

Comment:
looks like this file has a different formatting that the rest of the files - is this intentional?

Reply:
Good catch. This file wasn't linted and I did not press the Format Document button, so it seems I just forgot it. Will fix it.